server/test/user.test.js

```diff
@@ -91,6 +91,98 @@ describe("User test", function () {
     });
   });
 
+  describe("Get data from several users", function () {
+    let token;
+    before(function (done) {
+      // Authenticate
+      chai
+        .request(app)
```

Comment:
You could use the requestAPI function of `index.test.js` but ok ! 👌

Reply:
done()